app/views/repositories/show.html.erb

```diff
@@ -8,7 +8,11 @@
 
 <section id="show-repository">
 
+
   <section id="repository-photos">
+    <% if @repository.youtube_link %>
```

Comment:
I think writing `if @repository.youtube_link.present?` is what you would want to use here. When I tried deleting the url, it did not remove this section from the repository 'show' page. But try it for yourself after fixing the `update` and see which one works better.

Reply:
Well observed. Thanks

Reply:
Everything is working now.